src/Propel/Generator/Command/InitCommand.php

```diff
@@ -50,17 +50,20 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $this->getHelperSet()->set(new DialogHelper());
-
-        /** @var $dialog DialogHelper */
-        $dialog = $this->getHelper('dialog');
```

Comment:
Maybe we can move this code block into a private `createDialogHelper` method? :)